packages/components/src/mobile/link-settings/index.native.js

```diff
@@ -86,9 +86,9 @@ function LinkSettings( {
 	urlValue,
 	// Attributes properties
 	url,
-	label,
+	label = '',
 	linkTarget,
-	rel,
+	rel = '',
```

Comment:
These default values were changed to avoid unnecessary attribute updates when the prop transitioned from `undefined` to `''`. The latter is the initial state value used in this component.

The additional, unnecessary attribute updates made it difficult to correctly set the `linkDestination`, which is the focus of this PR. Specifically, having more than one update meant the second update did not have the context (previous `url`) required to set the correct `linkDestination`.